packages/byods-demo-server/src/views/data-source.hbs

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>{{title}}</title>
+    <link rel="stylesheet" type="text/css" href="/stylesheets/data-source.css" />
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;600;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <h1>Data Sources</h1>
+    <h2>Org ID: {{orgId}}</h2>
+    <div class="main-container">
+      <div class="data-container">
+        <div id="dataSourcesList" class="data-sources-list">
+          {{#each dataSources}}
+            <div class="data-source-card">
+              <h3>Data Source {{@index}}</h3>
+              <p><strong>ID:</strong> {{this.id}}</p>
+              <p><strong>Schema ID:</strong> {{this.schemaId}}</p>
+              <p><strong>Application ID:</strong> <span class="application-id">{{this.applicationId}}</span></p>
+              <p><strong>Status:</strong> <span class="status {{this.status}}">{{this.status}}</span></p>
+              <p><strong>Created By:</strong> {{this.createdBy}}</p>
+              <p><strong>Created At:</strong> {{this.createdAt}}</p>
+              <div class="card-buttons">
+                <button class="edit-button" onclick="editDataSource('{{this.id}}')">Edit</button>
+                <button class="delete-button" onclick="deleteDataSource('{{this.id}}')">Delete</button>
+                <button class="refresh-button" onclick="refreshJWSToken('{{this.id}}')">Refresh Token</button>
+              </div>
+            </div>
+          {{/each}}
+        </div>
+      </div>
+    </div>
+
+    <!-- Plus Button for Adding New Data Source -->
+    <button id="addDataSourceButton" class="add-button" onclick="openAddForm()">+</button>
+
+    <!-- Edit/Add Data Source Modal -->
+    <div id="editDataSourceModal" class="modal">
```

Comment:
This is for the modal window. I felt it would be a good user experience if we showed this.
This file also holds all the UI for the data source related part of the SDK.